examples/pymc3_howto/custom_distribution.ipynb

```diff
@@ -0,0 +1,471 @@
+{
```

Comment:
<p>There are two extra details that are worth mentioning:</p><p> </p><ol><li>For continuous distributions you also have to define the default transform, or inherit from a more specific class like <code>PositiveContinuous</code>  which specifies what the default transform should be. </li><li>On V3 you are supposed to also specify at least&nbsp;one ”default value" for the distribution during <code>init</code>  such as <code>self.mode</code>,  <code>self.median</code>  and <code>self.mean</code>  (the latter only for continuous distributions). This is used by some samplers or other compound distributions</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/185/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
Thank you for pointing these out! I'll add these notes to the text block. I also realized the reason I needed to set a `testval` when sampling was because I hadn't specified a "default value" for the distribution, so I can take that out now.